woocommerce/woocommerce-gutenberg-products-block
# Add quantity limit at product level

Moves the quantity selector limit from global to product level so that we can enforce a max quantity if stock level is lower than the limit. This prevents stock errors due to customers adding too much to the cart.

Fixes #2251

### How to test the changes in this Pull Request:

1. Give a product stock level 10, no backorders
2. Give a product stock level 10, with backorders
3. Make a product sold individually

Add each product to the cart and see if appropriate maximum is used.
